lte/gateway/deploy/roles/trfserver/files/traffic_server.py

```diff
@@ -587,6 +589,7 @@ def _run_iperf3(self, results_buffer, iperf):
         # Make the iperf3 call and spin off the subprocess
         self._server.log.debug('Running iperf3 command: %s', ' '.join(params))
         with subprocess.Popen(params, stdout=subprocess.PIPE) as proc:
+            started_iperf_instances[0] += 1
```

Comment:
I don't think this is thread-safe in Python.
https://stackoverflow.com/questions/1717393/is-the-operator-thread-safe-in-python

We might want to build a counter with a lock and pass that to the threads instead of an array. I don't think Python has a built-in solution for that (something like Java's AtomicInteger).

Reply:
changed to thread safe approach